mappings/net/minecraft/block/SmallTranslucentBlock.mapping

```diff
@@ -0,0 +1 @@
+CLASS blb net/minecraft/block/SmallTranslucentBlock
```

Comment:
I don't really get the meaning of the "Small" part. The methods in this class were previously part of the `GlassBlock` and `StainedGlassBlock` classes, so it would probably be more suitable as some kind of `AbstractGlassBlock`.

Reply:
I may have really misread this, and I can't check right now, but isn't it extended by G**r**assBlock?

Reply:
Nope, definitely Glass

Reply:
Hmm, then I'm blind. I'll change it when I get home in a few hours.